ui/src/router/components/form/components/router_config/rule_card/RuleCard.js

```diff
@@ -62,42 +67,53 @@ export const RuleCard = ({
     onChange("routes")([...rule.routes]);
   };
 
+  // Hide last row when all routes are selected
+  const routeSelectionOptions = (rule.routes.length < routes.length ? [...rule.routes,  "_none_"]: rule.routes);
+
   return (
     <EuiCard
       className="euiCard--routeCard"
-      title=""
+      title={isDefault ? "Default Traffic Rule" : ""}
       description=""
       textAlign="left">
+      {!isDefault ? 
```

Comment:
Same comment as above for this and other parts of this file - can use `&&` in place of the ternary operator to write the conditional blocks more concisely, where `null` is the result of the else block.